iFixit/react-commerce
# Widen zoomed image in product page

closes #966 

### QA

1. Visit Vercel preview
2. Verify that the the zoom container is now taking all the space available in the right side of the screen (with a 24px space on all sides)
